### PR TITLE
[NEW] Show list of chats using the SDK (pure online)

### LINKED ISSUE #604: [NEW] Show list of chats using the SDK (pure online)

### PR DESCRIPTION
**Pull request by philipbrito**
Closes #604 

**TODO:**

- [x] Get room name
- [x] Get unread count
- [x] Get room avatar (room initials)
- [x] Get last message (if allowed by server settings -> simple check if content isn't null)
- [x] Get last message timestamp (if allowed by server settings -> simple check if content isn't null)
- [ ] Get system message (e.g  _sent an attachment_) -> Blocked by SDK

### CODE REVIEW

File: app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        val chatRoom = dataSet[position]
+        val chatRoomName = chatRoom.name
+
+        holder.chatName.textContent = chatRoomName
+
+        if (chatRoom.type == RoomType.ONE_TO_ONE) {
+            // TODO Check the best way to get the current server url.
+            val canonicalUrl = chatRoom.client.restUrl.toString()
```
Comment (rafaelks) [inline app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsAdapter.kt:33]: The server URL is a setting called `Site_Url`.

File: app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsAdapter.kt [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+        }
+
+        val totalUnreadMessage = chatRoom.unread
+        when {
+            totalUnreadMessage in 1..99 -> {
+                holder.unreadMessage.textContent = totalUnreadMessage.toString()
+                holder.unreadMessage.setVisibility(true)
+            }
+            totalUnreadMessage > 99 -> {
+                holder.unreadMessage.textContent = context.getString(R.string.msg_more_than_ninety_nine_unread_messages)
```
Comment (rafaelks) [inline app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsAdapter.kt:48]: 👍

File: app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsAdapter.kt (hunk 1 above)
Comment (philipbrito) [reply app/src/main/java/chat/rocket/android/chatrooms/ui/ChatRoomsAdapter.kt:33]: Thanks for notify @rafaelks 👍 
This is for both of us to know @luciofm ^^

File: app/src/main/java/chat/rocket/android/app/DateTimeHelper.kt [hunk 3]
```diff
@@ -1,8 +1,6 @@
 import android.content.Context
 import chat.rocket.android.R
-import org.threeten.bp.LocalDate
-import org.threeten.bp.LocalDateTime
-import org.threeten.bp.Period
+import org.threeten.bp.*
```
Comment (philipbrito) [inline app/src/main/java/chat/rocket/android/app/DateTimeHelper.kt:3]: Interesting... the auto-complete imports all `org.threeten.bp` package. We should avoid it

**Pull request merged**